### PR TITLE
Added warning handler for modifying contents of tests/wpt/css-tests/

### LINKED ISSUE #55: Warn on modifying any files in tests/wpt/css-tests/
The files in that directory are synced from upstream occasionally, and that process will overwrite any modifications since the last sync. We should warn about this. This should be very similar to the handler added in #54.

### PR DESCRIPTION
**Pull request by saurvs**
Fixes #55.

### CODE REVIEW

File: handlers/no_modify_css_tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+NO_MODIFY_CSS_TESTS_MSG = '''This pull request modifies the contents of
+'tests/wpt/css-tests/', which are overwriten occasionally whenever the
+directory is synced from upstream.'''
+
+class NoModifyCSSTestsHandler(EventHandler):
+    DIR_TO_CHECK = "tests/wpt/css-tests"
+
+    def on_pr_opened(self, api, payload):
+        for line in api.get_diff().split('\n'):
+            if self.DIR_TO_CHECK in line:
```
Comment (jdm) [inline handlers/no_modify_css_tests/__init__.py]: Let's make a change like https://github.com/servo/highfive/commit/1c17b665e740b0eaff0e9083e4a6fd695eb93474 and ensure we're only checking the lines that start with `diff --git`.

**Comment by jdm** [comment]
With that small change this looks fine!

**Comment by saurvs** [comment]
Done!

**Comment by jdm** [comment]
Thank you!

**Pull request merged**